exoctk/contam_visibility/visibilityPA.py

```diff
@@ -16,7 +16,7 @@
 from astropy.table import Table
 from astropy.time import Time
 from bokeh.plotting import figure, ColumnDataSource
-from bokeh.models import HoverTool, ranges
+from bokeh.models import HoverTool

```

Comment:
@jaymedina We need this `ranges` import in order to set limits on the `bokeh` plots.